regression/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ add_subdirectory(contracts)
 add_subdirectory(goto-harness)
 add_subdirectory(goto-harness-multi-file-project)
 add_subdirectory(goto-cc-file-local)
+add_subdirectory(goto-cc-regression)
```

Comment:
Is there really no way this can fit into any of the existing collections of tests?

Reply:
Hi Martin, I spent some time this morning to make sure I can't get the test into another folder using the existing scaffolding.

I can't find a way myself, as this requires a very specific sequence in the `chain.sh`. Everyone of these folders have a specific `chain.sh` file that acts as scaffolding for that folder. Changing them causes issues which I can't fix easily. 

Hence, I think it's best to leave it as it is, unless you strongly object to that.